lms/templates/certificates/_accomplishment-rendering.html

```diff
@@ -75,10 +75,10 @@ <h3 class="accomplishment-orgs-title sr-only">${_("Supported by the following or
                         </div>
                     </li>
 
-                    % if certificate_data and certificate_data.get('org_logo_path', ''):
+                    % if organization_logo:
                     <li class="wrapper-organization">
                         <div class="organization">
-                            <img class="organization-logo" src="${static.url(certificate_data['org_logo_path'])}" alt="${platform_name}">
+                            <img class="organization-logo" src="${organization_logo.url}" alt="${platform_name}">
```

Comment:
Sorry, this is an architectural question. Where are these logo's going to be stored, in practice? A public S3 bucket?